doc/user/content/get-started.md

```diff
@@ -138,172 +132,307 @@ The first step is to declare where to find and how to connect to the sample Kafk
-            SELECT item_name, item_category, revenue
-            FROM item_summary_5min
-    )) TO STDOUT;
+    CREATE MATERIALIZED VIEW item_purchases AS 
+    SELECT 
+        item_id, 
+        SUM(quantity) AS items_sold, 
+        SUM(purchase_price) AS revenue, 
+        COUNT(id) AS orders, 
+        MAX(created_at::timestamp) AS latest_order 
+    FROM purchases 
+    GROUP BY item_id;
     ```
 
-    To cancel out of the subscription, press **CTRL+C**.
+1. Switch to the `default` cluster and query the materialized view:
 
-## Break something!
+    ```sql
+    SET CLUSTER = default;
+    ```
+    <p></p>
 
-1. From a different terminal window, open a new connection to Materialize.
+    ```sql
+    SELECT * FROM item_purchases LIMIT 10;
+    ```
+
+    ```nofmt
+     item_id | items_sold |  revenue  | orders |    latest_order
+    ---------+------------+-----------+--------+---------------------
+        1024 |       1300 |    600899 |    438 | 2023-04-20 05:52:10
+         768 |       1180 |  344465.6 |    401 | 2023-04-20 05:58:10
+         513 |       1393 |    234024 |    455 | 2023-04-20 05:55:27
+         256 |       1176 | 435108.24 |    390 | 2023-04-20 05:56:38
+         512 |       1233 | 150758.91 |    405 | 2023-04-20 05:56:48
+        1280 |       1302 | 488822.88 |    421 | 2023-04-20 05:57:11
+        1536 |       1183 | 181152.79 |    401 | 2023-04-20 05:58:14
+        1792 |       1204 | 203042.56 |    403 | 2023-04-20 05:59:34
+           1 |       1205 |  260304.1 |    414 | 2023-04-20 05:58:46
+         257 |       1172 | 469010.96 |    383 | 2023-04-20 05:53:31
+    (10 rows)
+
+    Time: 413.154 ms
+    ```
+
+    As you can see, the materialized view can be accessed from the `default` cluster as well. This is because, unlike an index, the **the results of a materialized view are persisted in durable storage** and incrementally updated as new data arrives. Because the storage layer is shared across all clusters, materialized views allow you to share data between different environments.
+
+    Also note that Materialize can output data to an external [sink](/overview/key-concepts/#sinks) only from a materialized view. It is not possible to output to a sink from a view or index.
 
-1. Add an additional replica to the `quickstart` cluster:
+## Step 5. Survive failures
+
+Earlier, when you created your clusters, you gave each cluster one [replica](/overview/key-concepts/#cluster-replicas) (that is, one physical resource). For the `ingest` cluster, that's the max number of replicas allowed, as clusters for sources can have only one replica. For the `compute` cluster, however, you can increase the number of replicas for greater tolerance to replica failure. 
+
+Each replica in a non-source cluster is a clone, doing the same computation and holding the same results in memory. This design provides Materialize with active replication, and so long as one replica is still reachable, the cluster continues making progress.
```

Comment:
I would say "is a logical clone", since the replicas can be different sizes (I'd also add that they can be different sizes). This is cool and underrated since not every database can do that - many other systems only do replication where they have to be physically identical as well).

Reply:
Done. I added a "Scale up or down" step to show one reason to add/remove replicas of different sizes.